xtest/test_tdfs.py

```diff
@@ -146,15 +155,43 @@ def test_tdf_spec_target_422(
         encrypt_sdk,
         "ztdf",
         tmp_dir,
+        scenario="target-422",
         target_mode="4.2.2",
     )
     assert os.path.isfile(ct_file)
+
+    manifest = tdfs.manifest(ct_file)
+    assert manifest.payload.isEncrypted
+
+    looks_like_422(manifest)
+
     fname = os.path.basename(ct_file).split(".")[0]
     rt_file = f"{tmp_dir}test-{fname}.untdf"
     decrypt_sdk.decrypt(ct_file, rt_file, "ztdf")
     assert filecmp.cmp(pt_file, rt_file)
 
 
+def looks_like_422(manifest: tdfs.Manifest):
+    if manifest.schemaVersion:
+        assert manifest.schemaVersion == "4.2.2"
```

Comment:
if it's 4.2.2 schema version should be empty, otherwise the go and java sdks will think it's not hexed